utilities/lua/rocks_lua_compaction_filter.cc

```diff
@@ -134,7 +134,7 @@ bool RocksLuaCompactionFilter::Filter(int level, const Slice& key,
 }
 
 const char* RocksLuaCompactionFilter::Name() const {
-  std::string name = "";
+  name_ = "";
```

Comment:
I'm approving it to unblock the build. But I think we shouldn't recalculate the name if we already have it in `name_`.